### DISCUSSION TITLE
SNUB Hook common problems and fixes.

### DISCUSSION THREAD

**Discussion by udkpg** [psvks/NotifyControl#18] [Announcements]
**SNUB hook** by **udkpg** ❤️
RECENT Releases: [here](https://github.com/udkpg/snub-api/releases)

FAQ:

1. **Where can I get snub hook to the latest version.**
  You  can get snub hook to the latest version by initializating the loadstring below.
   ```css
   loadstring(game:HttpGetAsync("https://udkpg.github.io/snub/snub_version/snub_ version you can see it in the original github.lua"))()
   ```
   Replace ` version you can see it in the original github` for the  version shown in the github
   
2. **Is snub hook still working?**
  You can check that yourself by entring [here](https://udkpg.github.io/snub/snub_version/snub_changelog/latestConsole.log)  
  It updates every week. So do not panic if the console says FAIL. you can still use it.
  
3. **Why snub hook has a strange script.**
  Snub hook uses RMD to encrypt  the script. We use this because it has more support.
  
***

Common problems and fix:

1. **SNUB Hook not executing.**
   **SNUB Hook is not executing because:**
       You need a supported executor shown in read me in the original github
       You can still see it clicking [here](https://github.com/udkpg/snub-api/blob/main/README.md)
2. **SNUB Hook needs to update**
      This is a really common problem. You just need to update it. like 
      ```css
        -- Just update the loadstring in https://github.com/udkpg/snub-api/releases
      ```
      This is the way to update it.
3. **SNUB Hook functions not working properly.**
   This doesnt has a fix yet. You may contact us or support team
   
   ***
   
   
   
   
   Thanks for reading. With MIT License. (c) 2022
   
  **BIG Note:** This fixes and problems wont apply to SNUB by psvks and more supported.

**Comment by NotifyControl** [comment]
## How to contact snub hook

  @udkpg DM him for support.
  
+rep 👍

**Reply by tarraqusuper** [reply]
[message deleted by moderator]

**Reply by NotifyControl** [reply]
@tarraqusuper We have moderated your comment. Next time make it more friendly.

**Comment by Aethetical** [comment]
Why is snub hook here?

ngl +rep 👍 essay lma

**Comment by SupremmeNig** [comment]
VOUCH WORKS FINE DUDE

**Comment by VannesaUWk** [comment]
vouch brotha

**Comment by NotifyControl** [comment]
SNUB CLASSIC IS NOT SUPPORTED ANYMORE FOR THE LATESTS VERSIONS OF SNUB.
